README.md

```diff
@@ -31,152 +42,177 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
+```swift
 
-**Note:** The user currently has to import both targets in their code to use entities in both Core and Publisher (or Subscriber). In the future, we may expose both `AblyAssetTrackingCore` and `AblyAssetTrackingPublisher` through one target using `@_exported`, so users only need to import one module. Similarly, a new target that joins `AblyAssetTrackingCore` and `AblyAssetTrackingSubscriber` can be created in the future. 
+let connectionConfiguration = ConnectionConfiguration(authUrl: authUrl, clientId: clientId)
 
-### Running tests locally
+```
 
-- Install fastlane by running `gem install fastlane`
-- Running in Xcode: Xcode can automatically generate schemes based on the Swift Package. 
-  - To run tests for a specific test target, select the test target scheme (e.g. `AblyAssetTrackingCoreTests`).
-  - Press ⌘U or click `Product` > `Test`.
-  - Xcode can generate test coverage (go into the scheme's test settings and enable it).
-- Running using Fastlane: 
-  - To run all tests, run `fastlane test_all`
-  - To run only one target, run `fastlane test_target_name`, where test_target_name can be `test_core`, `test_internal` or other test lanes are defined in `./Fastfile`.
+## Resources
+- [Asset Tracking Documentation](https://ably.com/documentation/asset-tracking)
+- [Ably Documentation](https://ably.com/documentation)
 
-### Building / viewing Jazzy documentation locally
-- Run `. ./jazzy/build.sh`
-- Open `docs/index.html`
+## Feature Support
+iOS Client Library [feature support matrix](https://ably.com/documentation/asset-tracking).
 
-### Coding Conventions and Style Guide
+## Support, feedback and troubleshooting
+Please visit http://support.ably.com/ for access to our knowledgebase and to ask for any assistance.
 
-- The SDKs are written in Swift, however they still have to be compatible for use from Objective-C based apps.
-- Favor Protocol Oriented Programming with Dependency Injection when writing any code. We're unable to create automatic mocks in Swift, so it'll be helpful for writing unit tests.
-- SwiftLint is integrated into the project. Make sure that your code does not add any SwiftLint related warning.
-- Please remove default Xcode header comments (with author, license and creation date) as they're not necessary.
-- If you're adding or modifying any part of the public interface of SDK, please also update [QuickHelp](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/SymbolDocumentation.html#//apple_ref/doc/uid/TP40016497-CH51-SW1) documentation.
-- Docs are written for both Swift and ObjC
+You can also view the [community reported Github issues](https://github.com/ably/ably-asset-tracking-swift/issues).
 
-### Release Procedure
+To see what has changed in recent versions, see the [CHANGELOG](https://github.com/ably/ably-asset-tracking-swift/blob/main/CHANGELOG.md).
 
-#### Bumping the Version
+## Known Limitations
```

Comment:
This heading needs breathing space. Please ensure you have a single blank line above and below all headings. This is mainly for readability for those who are reading the markdown raw, as well as those editing these files.

There are other headings in this file which also need this spacing added.

Reply:
fixed here 8d9502ef2d7a61158697bf9ea7e5602796959f65